Dockerfile

```diff
@@ -57,11 +59,11 @@ RUN useradd -U -m superset && \
         unzip \
         python-dev && \
     apt-get clean -y && \
-    rm -rf /var/lib/apt/lists/* /tmp/google-chrome-stable_current.deb /tmp/chromedriver.zip
+    rm -rf /var/lib/apt/lists/* /tmp/google-chrome-stable_${CHROME_VERSION}.deb /tmp/chromedriver.zip
```

Comment:
i think you can just use a wildcard to remove the deb file, to not depends on the env var